src/env/node/git/localGitProvider.ts

```diff
@@ -2,6 +2,7 @@ import { readdir, realpath } from 'fs';
 import { homedir, hostname, userInfo } from 'os';
 import { resolve as resolvePath } from 'path';
 import { env as process_env } from 'process';
+import type { HostingServiceType } from '@gitkraken/gitkraken-components';
```

Comment:
I know we haven't done a great job of this in the existing code, and a refactor is probably due in other places where we did this, but just to prevent the existing problem from getting worse, please import this type in `src/git/models/graph` and then export it from there as an internal type to be used here (and in `githubGitProvider`), instead of directly importing from the component here. Would like to eventually have all imports from the library component in that one file.

Reply:
Here is the pattern I am referring to in that file (to isolate the imports)

![image](https://github.com/gitkraken/vscode-gitlens/assets/67011668/ca0f010b-ef91-4d22-83de-8d780ae33cf3)